### PR TITLE
Feature/MIDAZ-221

### PR DESCRIPTION
**Pull request by maxwelbm**
# Midaz Pull Request Checklist

## Pull Request Type
[//]: # (Check the appropriate box for the type of pull request.)

- [ ] Auth
- [ ] Infra
- [ ] Ledger
- [x] Mdz
- [ ] Transaction
- [ ] Pipeline
- [ ] Documentation

## Checklist
Please check each item after it's completed.

- [x] I have tested these changes locally.
- [x] I have updated the documentation accordingly.
- [x] I have added necessary comments to the code, especially in complex areas.
- [x] I have ensured that my changes adhere to the project's coding standards.
- [x] I have checked for any potential security issues.
- [x] I have ensured that all tests pass.
- [ ] I have updated the version appropriately (if applicable).
- [x] I have confirmed this code is ready for review.

## Additional Notes
[//]: # (Add any additional notes, context, or explanation that could be helpful for reviewers.)
## Obs: Please, always remember to target your PR to develop branch instead of main.

### CODE REVIEW

File: common/mmodel/portfolio.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+// Portfolio is a struct designed to encapsulate request update payload data.
+type Portfolio struct {
+	ID             string         `json:"id"`
+	Name           string         `json:"name"`
+	EntityID       string         `json:"entityId"`
+	LedgerID       string         `json:"ledgerId"`
+	OrganizationID string         `json:"organizationId"`
+	Status         StatusAllow    `json:"status"`
```
Comment (MartinezAvellan) [inline common/mmodel/portfolio.go]: Update you code, this values don't exists in status anymore.

File: common/mmodel/status.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package mmodel
+
+// StatusAllow structure for marshaling/unmarshalling JSON.
+type StatusAllow struct {
```
Comment (MartinezAvellan) [inline common/mmodel/status.go]: Use only status

File: components/mdz/internal/rest/.fixtures/portfolio_response_create.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    "id": "01931b44-6e33-791a-bfad-27992fa15984",
+    "name": "Leslie_Spencer Portfolio",
+    "entityId": "56db6bfc-12f3-4369-9929-07059220b96d",
+    "ledgerId": "01931b04-c2d1-7a41-83ac-c5d6d8a3c22c",
+    "organizationId": "01931b04-964a-7caa-a422-c29a95387c00",
+    "status": {
+        "code": "ACTIVE",
+        "description": "Teste Portfolio",
```
Comment (MartinezAvellan) [inline components/mdz/internal/rest/.fixtures/portfolio_response_create.json:9]: Update you code, this values don't exists in status anymore.

File: components/mdz/internal/rest/portfolio_test.go [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+	metadata := map[string]any{
+		"bitcoin": "3o5onPR55kL6ajk14dGL5Q1fEhAnvY",
+		"chave":   "metadata_chave",
+		"boolean": true,
+	}
+
+	input := mmodel.CreatePortfolioInput{
+		Name: name,
+		Status: mmodel.StatusAllow{
+			Code:           code,
```
Comment (MartinezAvellan) [inline components/mdz/internal/rest/portfolio_test.go]: Update you code, this values don't exists in status anymore.

**Review by qnen — APPROVED** [review]

**Review by MartinezAvellan — APPROVED** [review]

**Pull request merged**